modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/model/extension/xml/MacroExpansionModuleModel.java

```diff
@@ -240,6 +263,17 @@ private void macroExpandGlobalElements(List<ComponentModel> moduleComponentModel
     });
   }
 
+  private boolean existOperationThatUsesImplicitConfiguration() {
+    return applicationModel.getRootComponentModel().getInnerComponents().stream()
+        .filter(componentModel -> componentModel.getIdentifier().getName().equals(FLOW_ELEMENT_IDENTIFIER)
+            || componentModel.getIdentifier().getName().equals(SUBFLOW_ELEMENT_IDENTIFIER))
+        .flatMap(componentModel -> componentModel.getInnerComponents().stream())
```

Comment:
Won't this miss the cases where the use of the xml sdk operation is inside a scope, like an async or try scope?